workflow/README.md

```diff
@@ -310,6 +317,53 @@ To use Snowflake as a data source container, you need to configure Snowflake usi
 * [RAI Integration for Snowflake: Quick Start for Users](https://docs.relational.ai/preview/snowflake/quickstart-user)
 ### Configure Snowflake Container
 Snowflake container configuration is defined in this section: [Snowflake container example](../cli/README.md#snowflake-container-example). 
+## Source Partitioning
+Workflow manager supports 2 types of source partitioning: date partitioning and chunk partitioning. Sources can be date partitioned and chunk partitioned at the same time.
+All chunk partitioned source have `isChunkPartitioned: true` and all date partitioned source have `isDatePartitioned: true` in the [Configure Sources](#configure-sources) step.
+### Partition Source Naming
+Partitions should meet the criteria described below otherwise RAI Workflow Manager will not be able to load data from the source.
+#### Date Partitioned Source
+Sources partitioned by date should have a date in the file path or name. Date should be in the format `yyyymmdd`.
+Regexp for date partitioned source must have `<date>` group and must be defined in `part_resource_date_pattern` relation. Example:
+```rel
+def part_resource_date_pattern = "^(.+)/data_dt=(?<date>[0-9]+)/(.+).(csv|json|jsonl)$"
+```
+#### Chunk Partitioned Source
+Chunk partitioned sources should have a chunk index in the file name or path. Chunk index should be `int` value.
+Regexp for chunk partitioned source must have `<shard>` group and must be defined in `part_resource_index_pattern` relation. Example:
+```rel
+def part_resource_index_pattern = "^(.+)/part-(?<shard>[0-9])-(.+).(csv|json|jsonl)$"
```

Comment:
we can also add a `<rai_shard>` point, basically saying we allow two shard formats, one of which is expected to be what RAI exports when paritions (`_[0-9]*.csv`)